db/migrate/20220203130519_fix_visits_column_name.rb

```diff
@@ -1,4 +1,4 @@
-class FixColumnName < ActiveRecord::Migration[6.1]
+class FixVisitsColumnName < ActiveRecord::Migration[6.1]
   def change
     rename_column :visits, :visit_id_id, :visit_id
   end
```

Comment:
do not edit migrations if they were pushed to the repository
in case of simple renaming it won't break anything
but still, you should remember not to do it